odl/tomo/geometry/parallel.py

```diff
@@ -713,31 +848,71 @@ def det_axes_init(self):
         return self.detector.axes
 
     def det_axes(self, angles):
-        """Return the detector axes tuple at ``angle``."""
+        """Return the detector axes tuple at ``angles``.
+
+        Parameters
+        ----------
+        angles : `array-like`
+            Euler angles in radians describing the rotation of the detector.
+            If multiple angle pairs (or triplets) are given, they must be
+            stacked along axis 0.
+
+        Returns
+        -------
+        axes : tuple of `numpy.ndarray`'s
```

Comment:
I think i mentioned this above, but if possible try to make this a large ndarray. Is there a reason we want a tuple?